tls/s2n_security_policies.h

```diff
@@ -25,12 +25,58 @@
 /* Kept up-to-date by s2n_security_policies_test */
 #define NUM_RSA_PSS_SCHEMES 6
 
+/**
+ * Note that s2n-tls is not RFC compliant with
```

Comment:
We actually have a very specific way of quoting the rfc in s2n-tls, see [this format](https://github.com/aws/s2n-tls/blob/main/tls/extensions/s2n_client_renegotiation_info.c#L86). You can also use it to signify[ non-compliance](https://github.com/aws/s2n-tls/blob/main/tls/extensions/s2n_client_max_frag_len.c#L58).

Additionally I think the comment before the security policy should explain what a security policy is. Your comments here really only apply to two fields on the security policy.

But otherwise this looks really helpful!

Reply:
We have a specific way of quoting for compliance. These comments are NOT about compliance. Putting them here is not correctly labeling where we comply with the different extension requirements.

Reply:
I think you might be looking at an old revision? I at least followed the format that `maddeleine` linked, although I might not have gotten it 100% right.

Reply:
And do we document our RFC quoting format anywhere? I couldn't find anything in the repo.